pkg/server/server.go

```diff
@@ -378,29 +378,36 @@ func (s *Server) SendVerificationLink(c *gin.Context, email string) {
 }
 
 // VerifyEmail marks a user email as verified
-func (s *Server) VerifyEmail(c *gin.Context) {
+func (s *Server) VerifyEmail(c *gin.Context, redirectURL string) {
 	jwtUser, exists := c.Get(types.JWTUserCredentialsKey)
 	if !exists {
```

Comment:
This case may not arise as you are setting the key in line no. 60 of email.go. Is the understanding correct?

Reply:
Yes @avishnu . The understanding is totally correct. I have added this check in case we need to call this method from some other place too where we do not set this key.